include/swift/AST/Builtins.def

```diff
@@ -994,6 +994,12 @@ BUILTIN_MISC_OPERATION_WITH_SILGEN(AutoDiffProjectTopLevelSubcontext, "autoDiffP
 // autoDiffAllocateSubcontext: (Builtin.NativeObject, Builtin.Word) -> Builtin.RawPointer
 BUILTIN_MISC_OPERATION_WITH_SILGEN(AutoDiffAllocateSubcontext, "autoDiffAllocateSubcontext", "", Special)
 
+// autoDiffCreateLinearMapContext2: (T.Type) -> Builtin.NativeObject
```

Comment:
Some bike-shedding: maybe rename these to `autoDiffCreateLinearMapContextWithType` etc.?